### PR TITLE
Remove ssh_key_data fix, handled in runner now

### PR DESCRIPTION
**Pull request by AlanCoding**
##### SUMMARY
Making this PR for purposes of testing.

This does a complete 180 from https://github.com/ansible/awx/pull/13102, and removes the hack as opposed to expanding its scope. Since the runner fix is in `unstream_dir`, it should be active in the needed context.

This makes use of the fix https://github.com/ansible/ansible-runner/pull/1133

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - API

### CODE REVIEW

**Comment by AlanCoding** [comment]
@fosterseth I haven't forgotten about this. Just got delayed due to flake when trying to run integration tests. Restarted just now :crossed_fingers:

**Comment by AlanCoding** [comment]
tests are good

File: requirements/requirements_git.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 git+https://github.com/ansible/system-certifi.git@devel#egg=certifi
 # Remove pbr from requirements.in when moving ansible-runner to requirements.in
-git+https://github.com/ansible/ansible-runner.git@devel#egg=ansible-runner
+git+https://github.com/fosterseth/ansible-runner.git@fix_ssh_key_data_pipe_reading#egg=ansible-runner
```
Comment (AlanCoding) [inline requirements/requirements_git.txt]: We can undo this when the PR is merged.

**Review by fosterseth — APPROVED** [review]

**Comment by AlanCoding** [comment]
I un-did the change to our git requirements. The runner change merged a few days ago, so this should be workable now.

**Comment by AlanCoding** [comment]
Finally confirmed that hang on production instances from before the runner fix, and confirmed it's still fixed with this change. Merging soon.

**Pull request merged**